blank-canvas-blocks/block-templates/page.html

```diff
@@ -0,0 +1,6 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
```

Comment:
My assumption is that the default post (`single`) + `page` templates default do not have a header or footer. Should we remove the header, nav, and footer template parts from these?

Reply:
I would think if any would/wouldn't all wouldn't.  I'm impartial as to which the default is.

I think if there isn't a template part defined then it also wouldn't define that 'zone' (has another name) so perhaps there should be an "empty" template part that is the default in those spaces so that the spaces are defined?  
Does that make sense to do? That's my only argument of having SOMEthing there.

Reply:
I think the default for this theme is not to have them. As the default for Seedlet would be to have them, being the main difference between them (besides font families and color configurations, etc).

The way I was originally thinking about this, what made sense to me is to have 2 different versions of the single/page template, one with the header and footer and one without, as you may want to make use of both on your site depending on the case. And that will also be the case for the children, where you will want to have the option of no header/footer too, but the defaults will be reversed.